src/helpers.ts

```diff
@@ -113,9 +118,9 @@ function checkContainerType(container) {
     )
   }
 
-  function getTypeName(object) {
+  function getTypeName<T>(object: T) {
```

Comment:
nit:
```suggestion
  function getTypeName(object: unknown) {
```
Is equivalent. We don't use the generic parameter anyway.